include/envoy/http/protocol.h

```diff
@@ -10,5 +11,19 @@ namespace Http {
 enum class Protocol { Http10, Http11, Http2 };
 const size_t NumProtocols = 3;
 
+inline std::string getProtocolString(const Protocol& p) {
```

Comment:
Already implemented here: https://github.com/envoyproxy/envoy/blob/master/source/common/access_log/access_log_formatter.cc#L33

Please consolidate (and use the string in access log formatter to be consistent).

Reply:
consolidated. Thanks.